src/vm/CMakeLists.txt

```diff
@@ -264,6 +264,9 @@ set(VM_SOURCES_WKS
     weakreferencenative.cpp
 )
 
+list(APPEND VM_SOURCES_WKS 
```

Comment:
Oops sorry this was the reason for build break on mac and freebsd. This needs to be conditioned on linux and windows